app/src/main/java/de/westnordost/streetcomplete/data/osm/osmquests/OsmQuest.kt

```diff
@@ -8,8 +8,8 @@ import de.westnordost.streetcomplete.data.osm.mapdata.ElementType
 import de.westnordost.streetcomplete.data.osm.mapdata.LatLon
 import de.westnordost.streetcomplete.data.quest.OsmQuestKey
 import de.westnordost.streetcomplete.util.measuredLength
-import de.westnordost.streetcomplete.util.pointOnPolylineFromEnd
```

Comment:
This util function can also be deleted, since it's only used here.

Then, maybe even remove the `fromEnd` parameter from the private `pointOnPolyline` function there, rename it to `pointOnPolylineFromStart` and make it public?

Reply:
> This util function can also be deleted in SphericalEarthMath.kt, since it was only used here.

Rather not, the util functions in SphericalEarthMath are ... well, you never know when you might need them. They are quite generic.